### ISSUE TITLE
Truncate long link in umb-node-preview

### ISSUE THREAD

**Issue by bjarnef** [rasmusjp/umbraco-multi-url-picker#65]
When adding a long link and no title is set, could the name in `umb-node-preview` be truncated?

![image](https://user-images.githubusercontent.com/2919859/32842345-1d7a2040-ca1d-11e7-8033-8b4d899509c1.png)

**Comment by rasmusjp** [comment]
Not sure if this still is a problem or not, but I guess this is also a problem with other usages of `<umb-node-preview>` e.g. with MNTP if so it's something that should be fixed in UImbraco Coew

**Comment by bjarnef** [comment]
@rasmusjp yes, I added a fix for this some time ago https://github.com/umbraco/Umbraco-CMS/pull/2327
